workspaces/diff-engine/lib/index.d.ts

```diff
@@ -8,4 +8,40 @@ export function spawn({
   input: Duplex;
   output: Duplex;
   error: Duplex;
+  result: Promise<void>;
 };
+
+export interface DiffEngineError extends Error {
```

Comment:
the formatting seems messed up here but nbd

Reply:
Ahh, my editor was set to render tabs at 2 wide, so never noticed they got mixed. Github's 8 wide spaces to the rescue (do people even ever use that?). Fixing

Reply:
Fixed in https://github.com/opticdev/optic/pull/448/commits/23c5c025e42ff8507fc20c1e50fc8fd9aa07c3d2